src/Sentry.AspNetCore/DefaultUserFactory.cs

```diff
@@ -36,20 +36,21 @@ internal class DefaultUserFactory : IUserFactory
             // Identity.Name Reads the value of: ClaimsIdentity.NameClaimType which by default is ClaimTypes.Name
             // It can be changed by the application to read a different claim though:
             var name = principal.Identity?.Name;
-            if (name != null && username != name)
+            if (name is not null && username != name)
```

Comment:
It's fine to do the refactor in this change, lets go all the way then for consistency sake:

```suggestion
            if (name is not null && username is not name)
```